lib/cn/cn_tree_internal.h

```diff
@@ -133,21 +133,21 @@ struct cn_tree {
     uint                 ct_lvl_max;
     struct cn_samp_stats ct_samp;
 
-    HSE_ALIGNED(SMP_CACHE_BYTES) union {
-        struct sp3_tree sp3t;
+    union {
```

Comment:
Does this have to be a union if it has one member?

Reply:
Prob not, @alexttx ???